src/main/java/com/launchdarkly/client/RedisFeatureStoreConfig.java

```diff
@@ -0,0 +1,205 @@
+package com.launchdarkly.client;
+
+import org.apache.http.client.utils.URIBuilder;
+import redis.clients.jedis.JedisPoolConfig;
+import redis.clients.jedis.Protocol;
+
+import java.net.URI;
+import java.net.URISyntaxException;
+import java.util.concurrent.TimeUnit;
+
+/**
+ * This class exposes advanced configuration options for the {@link com.launchdarkly.client.RedisFeatureStore}.
+ * Instances of this class must be constructed with a {@link Builder}
+ */
+public class RedisFeatureStoreConfig {
```

Comment:
General thought on the structure of this-- it doesn't seem generally useful to have a handle to the  `RedisFeatureStoreConfig` (unlike the `LDClient` config object, which is passed around internally for other reasons). The pattern as you have it here is to use the builder to create the `Config` object, then immediately pass the output of the builder to the `RedisFeatureStore`'s ctor. What if we just had a `RedisFeatureStoreBuilder`, and had a `build()` method that created a `RedisFeatureStore`? 

That seems to eliminate one unnecessary extra step. It might make testing a little tricker though? What are your thoughts?